### ISSUE TITLE
Missing loss mask with hierarchical action space in behavior cloning script?

### ISSUE THREAD

**Issue by phython96** [openai/Video-Pre-Training#26]
Hi, I found that finetuning foundation-model-1x starts with a high loss (14.0) using find-cave contractor data.
As described in section E.3, when using hierarchical action space, loss for the secondary mouse movement should be masked during training. 
However, I didn't find the implementation in behavior_cloning.py, which may be the reason why the fintuning loss is large.  
Looking forward to your reply, thanks.

**Comment by Miffyli** [comment]
Hey! Sorry for the delay; I have been OOF :) 

Hmm can you pinpoint the phrase about "secondary mouse movement"? The action space should capture mouse movement regardless of the task/setup.

The high loss is also probably due to to different training setup; due to memory constraints, the BC example trains with very small sequences lengths (and one sample at a time). The behaviour of the players in the find-cave (or other BASALT task) is also significantly different from general Minecraft gameplay, so I am not that surprised the loss starts high.
